bazaarvoice/emodb
# PD-169594_tests for PUT /bus/1/{subscription}

## Github Issue #

[PD-169594](https://github.com/bazaarvoice/emodb/issues/PD-169594)

## What Are We Doing Here?

Tests for PUT /bus/1/{subscription} have been added in postman

## How to Test and Verify

1. Check out this PR
2. Import new postman collection into postman, Run collection
3. All tests should be passed

## Risk

### Level 

`Low`

### Required Testing

`Manual`. 
### Risk Summary

There shouldn't be any risk as these changes as this change is only adding a new tests

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
